src/alda/repl/AldaRepl.java

```diff
@@ -144,18 +148,11 @@ public void run() {
           history.append("\n");
 
           // If we're good, we should check to see if we reset the instrument
-		  if (playResponse != null &&
-			  playResponse.score != null &&
-			  playResponse.score.currentInstruments.length > 0) {
-			  // If we have multiple instruments, pick the first one.
-			  String instrument = playResponse.score.currentInstruments[0];
-			  if (instrument.length() > 0) {
-				  // Pick the first letter of the instrument
-				  promptPrefix = Character.toString(instrument.charAt(0));
-			  } else {
-				  promptPrefix = "";
-			  }
-		  }
+          if (playResponse != null && playResponse.currentInstrument() != null) {
+            // If we have multiple instruments, pick the first one.
+            String instrument = playResponse.currentInstrument();
+            this.setPromptPrefix(instrument);
```

Comment:
In the Clojure Alda REPL we have now, we do something a little more complicated when displaying the current instruments:

- We set the prompt to the abbreviations of all of the current instruments, with slashes in between, e.g.: `p/t>` for a `piano` and a `trumpet`.

- If the instrument name contains dashes, we treat that as multiple "words" and the abbreviation is the first letter of each word, e.g. `midi-electric-piano-1` becomes `mep1`.

- If an instrument has a nickname, we use the nickname instead. For example, if you enter `trombone "foo": c`, the prompt reads `f>` for `foo`.

  These nicknames are stored as a map of nickname => list of instance IDs in the score map:

  ```clojure
  :nicknames
  {"foo" ("trumpet-KsU1s"),
   "strings" ("violin-A2i5P" "viola-YcVdU"),
   "strings.violin" ("violin-A2i5P"),
   "strings.viola" ("viola-YcVdU"),
   "baz" ("harmonica-4EOEB"),
   "bar" ("tuba-OttVs")},
   ```

It would be nice if  we can replicate this in the new REPL.